easier_net/sier_net.py

```diff
@@ -12,24 +12,26 @@
 
 from network import SierNet
 
-
-class SierNetEstimator:
+class SierNetEstimator: 
     """
     This first a single sparse-input hierarchical network (SIER-net).
     """
     def __init__(
         self,
-        n_inputs: int,
-        input_filter_layer: bool,
-        n_layers: int,
-        n_hidden: int,
-        n_out: int,
-        full_tree_pen: float,
-        input_pen: float,
-        batch_size: int,
-        num_classes: int,
-        weight: list,
-        connection_pen: float = 0,
+        n_inputs=4,
```

Comment:
If you plan to determine `n_inputs` in `fit`, then I don't think you need this here?